README.md

```diff
@@ -1 +1,35 @@
-# CarbonQueryDatabase
\ No newline at end of file
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/11a3ucotxcl9746k/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/CarbonQueryDatabase-Toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/CarbonQueryDatabase_Toolkit/CarbonQueryDatabase_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build?definitionId=198&_a=summary)
```

Comment:
I think the last link also needs to be updated to CarbonQueryDatabase_Toolkit

Reply:
I believe it already is as the link takes you to the CarbonQueryDatabase_Toolkit.CheckCore page. Please confirm!

Reply:
You are correct - my mistake!